tests/regression/tests/REQUEST-922-MULTIPART-ATTACK/922130.yaml

```
@@ -0,0 +1,41 @@
+---
+meta:
+  author: "Ervin Hegedus"
+  description: Test Multipart/form-data
+rule_id: 922130
+tests:
+  - test_id: 1
+    desc: "Positive test: special character in multipart header"
```

Add the decoded contend (or at least a pseudo version of it) to the description, so its clear what the request looks like.
```suggestion
    desc: |
      Positive test: special character in multipart header.
      <decoded content>
```

---

Done